app/js/components/page/pageComponent.jsx

```diff
@@ -54,6 +55,19 @@ class PageComponent extends Component{
   }
 
   /**
+   * Clear all saved searches by deleting them from session storage
+   * @return {void} void
+   * @memberof SearchHistoryComponent
+   */
+  clearSearchHistory() {
+    const confirmDelete = confirm('Are you sure you want to delete?');
```

Comment:
to delete what?

Reply:
I've fixed it by making it more descriptive